_config.yml

```diff
@@ -73,33 +73,200 @@ defaults:
+          label: "Position"
+      - name: "email"
+        field:
+          element: "text"
+          label: "E-mail address"
+      - name: "github"
+        field:
+          element: "text"
+          label: "GitHub username"
+      - name: "employer"
+        field:
+          element: "text"
+          label: "Employer"
+    blog/_posts:
+      - name: "published"
+        field:
+          element: "hidden"
+          value: true
+      - name: "title"
+        field:
+          element: "text"
+      - name: "author"  # make this a multiselect
+        field:
+          element: "text"
+          label: "Author"
+      - name: "tags"  # make this a multiselect
+        field:
+          element: "text"
+          label: "Tags"
+    jobs/_posts:
+      - name: "published"
+        field:
+          element: "hidden"
+          value: true
+      - name: "title"
+        field:
+          element: "text"
+      - name: "author"  # make this a multiselect
+        field:
+          element: "text"
+          label: "Author"
+      - name: "tags"  # make this a multiselect
+        field:
+          element: "text"
+          label: "Tags"
+    products-tools/_posts:
+      - name: "published"
+        field:
+          element: "hidden"
+          value: true
       - name: "title"
         field:
           element: "text"
-          label: "title"
-      - name: "layout"
+      - name: "image-link"
+        field:
+          element: "text"
+          label: "Product or Tool URL"
+      - name: "image-source"
+        field:
+          element: "text"  # make this a multiselect
+          label: "Image file path"
+      - name: "image-alt"
+        field:
+          element: "text"
+          label: "Alt-text for the image"
+      - name: "image-height"
+        field:
+          element: "number"
+          label: "Image height in pixels"
+      - name: "image-width"
+        field:
+          element: "number"
+          label: "Image width in pixels"
+    projects/_posts:
+      - name: "published"
         field:
           element: "hidden"
-          label: "Layout"
-          value: "post"
-      - name: "categories"
+          value: true
+      - name: "title"
+        field:
+          element: "text"
+      - name: "author"  # make this a multiselect
+        field:
+          element: "text"
+          label: "Author"
+      - name: "tags"  # make this a multiselect
+        field:
+          element: "text"
+          label: "Tags"
+      - name: "type"
         field:
-          element: "multiselect"
-          label: "Add categories"
-          placeholder: "Choose categories"
+          element: "select"
+          label: "Project type"
           options:
-            - name: "Infrastructure"
-              value: Infrastructure
-            - name: "Energy Planning"
-              value: energy planning
-          alterable: true
-      - name: "tags"
+            - name: "ICT4D and Development Plannin"
```

Comment:
damn

Sent from my phone.
On Mar 18, 2016 4:19 PM, "Chris Natali" <notifications@github.com> wrote:

> In _config.yml
> <https://github.com/SEL-Columbia/SEL-Columbia.github.io/pull/98#discussion_r56716359>
> :
>
> >            options:
> > -            - name: "Infrastructure"
> > -              value: Infrastructure
> > -            - name: "Energy Planning"
> > -              value: energy planning
> > -          alterable: true
> > -      - name: "tags"
> > +            - name: "ICT4D and Development Plannin"
>
> Planning?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/SEL-Columbia/SEL-Columbia.github.io/pull/98/files/8fe2728acae8fcb2974b786bc838f4f8da1b0bce#r56716359>
>